packages/design-tokens/src/scss/generated/_typography.scss

```diff
@@ -29,6 +29,16 @@ $body-large-text-regular: (
     ),
 ) !default;
 
+$body-large-text-semibold: (
+    mobile: (
+        font-family: "'Inter', sans-serif",
+        font-size: 1.125rem,
+        font-style: normal,
+        font-weight: 500,
```

Comment:
Looks like [there should be added exception](https://github.com/lmc-eu/spirit-design-system/pull/714/files#diff-1122ffd4a11323cd6b8035fb246cb0e6af8db8e90012ce7582607fa66e5a185aR17) for `Inter` as well.

<img width="1868" alt="image" src="https://github.com/user-attachments/assets/82e8f2d0-d493-4928-8dd0-f5cdee5454c4">

Reply:
🙃 the title is not in the typography

Reply:
click on the text below and you will see that the semibold is 500.

Reply:
however, we somehow have bold sometimes 600 and sometimes 700, so we should fix that.